.storybook/preview.js

```diff
@@ -1,31 +1,20 @@
 import React from 'react';
-import { addDecorator, addParameters } from '@storybook/react';
-import { withKnobs } from "@storybook/addon-knobs";
-import { withThemePlayground } from 'storybook-addon-theme-playground';
-import { ThemeProvider } from "styled-components";
+import { Provider } from 'react-redux';
 
-import theme, { Theme } from '../client/theme';
+import ThemeProvider from '../client/modules/App/components/ThemeProvider';
+import configureStore from '../client/store';
 
-addDecorator(withKnobs);
+const initialState = window.__INITIAL_STATE__;
 
-const themeConfigs = Object.values(Theme).map(
-  name => {
-    return { name, theme: theme[name] };
-  }
-);
+const store = configureStore(initialState);
 
-addDecorator(withThemePlayground({
-  theme: themeConfigs,
-  provider: ThemeProvider
-}));
+export const decorators = [
+  (Story) => (
+      <Provider store={store}>
```

Comment:
If we are connecting to Redux here then maybe we want a way to modify the initial state on a per-story basis?  Just a thought.  That's something for a future PR.

Reply:
I'm not sure what Redux is used for but all components with stories needed it. It could be good to try and remove it as a dependancy from Storybook/Design System components 🤷🏽

Reply:
Yeah I actually agree. I think of Storybook as being for pure UI components which are not connected to Redux.  But the current `FileNode` and the `SocialAuthButton` components are connected and they have stories so that's why we need it for now.